content/en/data/programs.json

```diff
@@ -1163,40 +1276,50 @@
     ],
     "how_it_works": [
-      "Anyone who uses SNAP/EBT card (food stamps) can use Fresh Bucks Match! You’ll get $1 worth of Fresh Bucks for every $1 you spend on qualifying fruits and vegetables* at any participating Fresh Bucks retailer. Here’s how:",
+      "Anyone who buys fruits and vegetables with their SNAP/EBT benefits can get $1 worth of Fresh Bucks for every $1 you spend on fruits and vegetables at participating Fresh Bucks retailers.",
+      "-",
       "AT THE FARMERS MARKET:",
-      "1. Go to the market’s Information Booth and tell them you’d like Fresh Bucks",
+      "1. Go to the market’s Information Booth",
       "2. Swipe your EBT card for the amount you want to spend",
       "3. You’ll receive that amount in EBT tokens AND that same amount in Fresh Bucks dollars",
       "4. Shop for fruits and vegetables using your Fresh Bucks dollars",
-      "AT SEATTLE SAFEWAY STORES:",
-      "1. Buy at least $10 in qualifying fruits and vegetables* with your EBT card",
-      "2. Get a $5 coupon to spend on fruits and vegetables on your next shopping trip.",
+      "-",
       "AT NEIGHBORHOOD GROCERS AND FARM STANDS:",
-      "1. Purchase qualifying fruits and vegetables*",
+      "1. Purchase qualifying fruits and vegetables",
       "2. Pay with your EBT card",
       "3. Get 50% off your fruit and vegetable purchase"
     ],
-    "age_range": "Everyone",
-    "person_application": "Apply in-person at any Community Services Office. Use Washingon State's website to locate the nearest Community Services Office (https://www.dshs.wa.gov/office-locations?field_geofield_distance%5Bdistance%5D=100&field_geofield_distance%5Bunit%5D=3959&field_geofield_distance%5Borigin%5D=&field_office_type_tid%5B0%5D=9651).",
-    "application_info": "To use Fresh Bucks Match, you must receive Basic Food Assistance (SNAP/EBT/food stamps) (https://www.dshs.wa.gov/esa/community-services-offices/basic-food) through Washington State. If you do receive Basic Food Assistance, you do not need to apply to use Fresh Bucks at any participating retailer. If you do not receive Basic Food Assistance, here are your options. ",
-    "savings": 23,
-    "savingsByPeriod": "$23 / Month",
-    "time_to_apply": "1 Hour",
-    "time_hear_back": "3 Weeks",
-    "online_application": "https://www.washingtonconnection.org/home",
-    "application_form": "https://www.dshs.wa.gov/sites/default/files/FSA/forms/pdf/14-001.pdf",
+    "questions": {
+      "text": [
+        {
+          "category": "cell",
+          "value": "Call: (206) 684-2489"
+        },
+        {
+          "category": "email",
+          "value": "Email: freshbuck@seattle.org"
+        }
+      ]
+    },
+    "age_range": "Anyone who uses SNAP/EBT (food stamps)",
+    "person_application": "Apply in-person at any Community Services Office. Use Washington State's website to locate the nearest Community Services Office (https://www.dshs.wa.gov/office-locations).",
```

Comment:
Note: the links in person_application are not showing as links. (might want to make a separate pr up for that. but just wanted to make sure you all are aware of it)
<img width="423" alt="Screen Shot 2019-09-05 at 6 46 56 PM" src="https://user-images.githubusercontent.com/19806441/64395012-92f20e80-d00d-11e9-93cf-9111a1daf48b.png">